app/webpacker/components/Panel/pages/RunValidatorsPage/api/getCompetitionCount.js

```diff
@@ -0,0 +1,7 @@
+import { fetchJsonOrError } from '../../../../../lib/requests/fetchWithAuthenticityToken';
+import { viewUrls } from '../../../../../lib/requests/routes.js.erb';
+
+export default async function getCompetitionCount(startDate, endDate) {
+  const { data } = await fetchJsonOrError(viewUrls.competitions.countInRange(startDate, endDate));
+  return data?.count;
```

Comment:
Hm, now you're accessing the data already again. If only the count is so important, then why not just return it directly from the backend?
(Yes, numbers by themselves are valid JSON. Not everything has to be `{}` or `[]` on top level)